include/triton/Dialect/TritonGPU/IR/Dialect.h

```diff
@@ -152,6 +152,12 @@ void dumpHWLayout(RankedTensorType tensorType);
 // Return a string representation of the layout of the tensor.
 std::string getLayoutStr(RankedTensorType tensorType, bool useHWPointOfView);
 
+// Helper functions for getLayoutStr
+std::string getSharedLayoutStr(RankedTensorType tensorType,
```

Comment:
What is the purpose to expose these two lower level APIs here since the `getLayoutStr` would dispatch to one of them. Isn't `getLayoutStr` enough capture both of them?

Reply:
you're right; we probably don't need to export these

fixed